README.md

```diff
@@ -96,49 +96,18 @@ Note, that in the example above, the subscription must be in the `marketplace` (
 
 After an operator has been installed, to uninstall the operator you need to delete the following resources. Below we uninstall the `jaeger` operator as an example.
 
-Delete the `Subscription` in the `targetNamespace` that the operator was installed into. If created via the OpenShift OperatorHub UI, it will be named after the operator's packageName.
+Delete the `Subscription` in the namespace that the operator was installed into. For OKD, this is the `marketplace` namespace. Keeping to the above example subscription `jaeger`, we can run the following command to delete it from the command line:
```

Comment:
For OKD it is not `marketplace` but `openshift-marketplace`. i think the problem might be with the fact that the install example is creating the `Subscription` in the `marketplace` namespace. Now the `Subscription` can be anywhere. So changing that might make things easier.

Also, `Note, that in the example above, the subscription must be in the marketplace namespace` in line 93 no longer holds true, so that should be fixed too.

Reply:
That's not right, is it? For both OCP and OKD the namespace should be `openshift-marketplace`. It's just for upstream kube that it should be `marketplace`

Reply:
That's not right, is it? For both OCP and OKD the namespace should be `openshift-marketplace`. It's just for upstream kube that it should be `marketplace`

Reply:
Yeah sorry that was a typo on my end. Pushed a fix just now